lib/sonata/channelWeeklyAirdrop/calculateStakersReward.ts

```diff
@@ -0,0 +1,28 @@
+import { Staker } from "@/types/Stake";
+
+export default function calculateStakersReward(users: Staker[], totalAmount: number) {
+  const currentDate = new Date();
+
+  const userScores = users.map(user => {
+    let investedDays = 8;
+
+    if (user.earliest_week_stake) {
+      console.log('earliest_week_stake', user.earliest_week_stake);
+      const stakedDate = new Date(user.earliest_week_stake);
+      investedDays = Math.floor((currentDate.getTime() - stakedDate.getTime()) / (1000 * 3600 * 24));
+    }
+    const days = Math.min(investedDays, 8);
```

Comment:
what is `8` doing in this call?
- can you replace the `magic number` with a pre-defined variable?

Reply:
`days` = `daysStaked` right?

Reply:
open question, how are `unstaking` events considered in this calculation?

Reply:
As of now we are not taking into account `unstaking`.